odoo/models.py

```diff
@@ -2866,15 +2866,17 @@ def check_field_access_rights(self, operation, fields):
         def valid(fname):
             """ determine whether user has access to field ``fname`` """
             field = self._fields.get(fname)
-            if field and field.groups:
-                return self.user_has_groups(field.groups)
-            else:
-                return True
+            if field:
```

Comment:
Please don't, the attribute exportable shouldn't change the behavior of the `read`. It doesn't make any sense.

Also if the read of this kind of fields doesn't work, fix these fields (Or read explicitly the wanted fields). The job of `check_field_access_rights` is only to filter-out inaccessible (from security point of view) fields.

Reply:
Consider my PR as a way to point to the problem. I agree, that it's not really a solution